process/headerCheck/headerSignatureVerify.go

```diff
@@ -207,7 +207,8 @@ func (hsv *HeaderSigVerifier) VerifySignatureForHash(header data.HeaderHandler,
 }
 
 func (hsv *HeaderSigVerifier) getPrevHeaderInfo(currentHeader data.HeaderHandler) (data.HeaderHandler, []byte, []byte, []byte, error) {
-	sig, bitmap := currentHeader.GetPreviousAggregatedSignatureAndBitmap()
+	previousProof := currentHeader.GetPreviousProof()
+	sig, bitmap := previousProof.GetAggregatedSignature(), previousProof.GetPubKeysBitmap()
```

Comment:
check previousProof against nil before this

Reply:
done